lib/platforms.ts

```diff
@@ -43,6 +43,15 @@ export function platformToString(platform: Platform) {
   return map[platform];
 }
 
+export function environmentFromString(string: string) {
+  const map = {
+    'web': Environment.Web,
+    'desktop': Environment.Desktop,
+    'mobile': Environment.Mobile,
+  };
+  return (map as any)[string];
```

Comment:
Why the `as any`? Should be a way to do it without?

Reply:
Without it, an error is shown:

`Element implicitly has an 'any' type because expression of type 'string' can't be used to index type...` and `No index signature with a parameter of type 'string' was found on type...`

---

We can add a type to `map` to get rid of the error, like so:

```javascript
const map: Record<string, number> = {
    'web': Environment.Web,
    'desktop': Environment.Desktop,
    'mobile': Environment.Mobile,
};
return map[string];
```

Also, we'll need to update `platformFromString` function to remove the `(map as any)`.

Reply:
> We can add a type to map to get rid of the error

Yes, that's the correct solution.

You should also add context to this PR so that in the future we can understand why we added this.

Reply:
Updated 👍🏽